resources/views/auth/register.blade.php

```diff
@@ -24,56 +24,82 @@ class="flow bg-indigo-500 hover:bg-indigo-700 text-white font-bold py-2 px-4 rou
 
 
                 <div class="mt-5">
-                    <x-label for="name" value="{{ __('Name') }}"/>
+                    <x-label for="name" value="{{ __('Name') }}" class="after:content-['*'] after:text-red-500"/>
                     <x-input id="name" class="block mt-1 w-full" type="text" name="name" :value="old('name')" required
                              autofocus autocomplete="name"/>
                 </div>
 
                 <div class="mt-4">
-                    <x-label for="email" value="{{ __('Email') }}"/>
+                    <x-label for="email" value="{{ __('Email') }}" class="after:content-['*'] after:text-red-500"/>
                     <x-input id="email" class="block mt-1 w-full" type="email" name="email" :value="old('email')"
                              required
                              autocomplete="username"/>
                 </div>
 
                 <div class="mt-4">
-                    <x-label for="password" value="{{ __('Password') }}"/>
+                    <x-label for="password" value="{{ __('Password') }}" class="after:content-['*'] after:text-red-500"/>
                     <x-input id="password" class="block mt-1 w-full" type="password" name="password" required
                              autocomplete="new-password"/>
+
+                    <div id="password-rules" class="hidden text-sm text-gray-700 dark:text-gray-300 mt-2 pl-2">
+                        <p>password should be of the following format:</p>
```

Comment:
It's nitpicky but really trips me - Can you do it with a capital P? :D

Reply:
Oh yeah, my bad. I changed `Password.php` in vendor folder and it was not pushed, that's why you don't have any server side validation. I will fix it in a minute. :D